api/tests/profile/acceptance/application/share-profile-reward-route_test.js

```diff
@@ -0,0 +1,76 @@
+import {
+  createServer,
+  databaseBuilder,
+  expect,
+  generateValidRequestAuthorizationHeader,
+} from '../../../test-helper.js';
+
+describe('Profile | Acceptance | Application | Attestation Route ', function () {
+  let server;
+
+  beforeEach(async function () {
+    server = await createServer();
+  });
+
+  describe('POST /api/users/{userId}/profile/share-reward', function () {
+    describe('when profile reward exists and is linked to user', function () {
+      it('should return a success code', async function () {
+        // given
+        const userId = databaseBuilder.factory.buildUser().id;
+        const profileReward = databaseBuilder.factory.buildProfileReward({ userId });
+        const campaignParticipation = databaseBuilder.factory.buildCampaignParticipation();
+
+        await databaseBuilder.commit();
+        const options = {
+          method: 'POST',
+          headers: { authorization: generateValidRequestAuthorizationHeader(userId) },
+          url: `/api/users/${userId}/profile/share-reward`,
+          payload: {
+            data: {
+              attributes: {
+                profileRewardId: profileReward.id,
+                campaignParticipationId: campaignParticipation.id,
+              },
+            },
+          },
+        };
+
+        // when
+        const response = await server.inject(options);
+
+        // then
+        expect(response.statusCode).to.equal(201);
+      });
+    });
+  });
+
+  describe('when profile reward does not exist', function () {
```

Comment:
**suggestion** : Changer le it, dans le cas de test il existe, juste il n'est pas lié au user